adopt-a-dog/styling/stylesheet.css

```diff
@@ -0,0 +1,392 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body{
+  background-color: #e6e6e6;
+  width : 100%;
+  max-width: 1450px;
+  min-width: 500px;
+}
+/****** Header ******/
+.page-banner
+{
+  background-image: url("../images/christopher-harris-57366-unsplash.jpg");
+  background-size: 100% 500px;
+  background-repeat: no-repeat;
+  background-position: top;
+  color: white;
+  height: 300px;
+  width: 100%;
+  padding: 10px;
+  display : flex;
+  align-items :center;
+}
+.page-banner-title{
+  padding-left: 30%;
+}
+.page-banner-title h1{
+  color: #4d9900;
+  font-size: 1.5rem;
+  position: absolute;
+  left: 8%;
+  top: 150px;
+  }
+.logo-top{
+ width: 30%;
+ height: 15%;
+ max-width: 35rem;
+ min-width: 15rem;
+ min-height: 10rem;
+ position: absolute;
+ left: 100px;
+ top: 20px;
+}
+.shopping-icon
+{
+ width: 5%;
+ min-width: 0.8rem;
+ height: 1.5rem;
+ position: absolute;
+ top: 30px;
+ right: 180px;
+}
+.banner-price{
+ font-size: 1rem;
+ color: white;
+ position: absolute;
+ top: 35px;
+ right: 120px;
+}
+.checkout-button-top
+{
+ background-color: #66cc00;
+ outline: none;
+ border: none;
+ color: white;
+ font-size: 1rem;
+ width: 12%;
+ min-width: 6rem;
+ padding: 5px;
+ margin: 10px;
+ position: absolute;
+ top: 55px;
+ right: 80px;
+}
+/* Banner Navigation*/
+.nav-bar-container{
+  width : 100%;
+  padding-left: 15%;
+}
+.nav-bar{
+  padding: 0.5rem;
+  width: 40%;
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  list-style-type: none;
+}
+.nav-bar li::after{
+  padding-left: 20px;
+  padding-right: 20px;
+  content: "/";
+}
+.nav-bar li:last-child::after{
+  content: "";
+}
+.nav-bar li a{
+  text-decoration : none;
+  color: #003366;
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+.wrapper{
+  background-color: white;
+  width: 100%;
+  max-width: 1450px;
+  min-width: 500px;
+  padding-left: 15%;
+  padding-right: 15%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+.page-title{
+  font-size: 2rem;
+  color: #003366;
+}
+/* article = Introduction*/
+.hugging-img{
+  width:15rem;
+  height: 15rem;
+}
+.intro{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+   max-width: 1450px;
+  margin: auto 0;
+}
+.intro-content{
+  display: flex;
+  flex-direction: column;
+  width: 70%;
+  padding-left: 30px;
+}
+.intro-content p{
+  font-size: 1.3rem;
+  align-self: center;
+  font-family: 'Noto Sans SC', sans-serif;
+}
+/* picture flex ----Dogs HTML*/
+.picture-flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content:space-around;
+  width: 100%;
+  margin: auto 0;
+  margin-top: 15px;
+}
+.dog-info{
```

Comment:
You shouldn't have to use flexbox on the tiles. See if you can pull it out and still align things inside of the tiles without it.

Reply:
I made the suggested changes :).